Block/Category.php

```diff
@@ -101,13 +105,12 @@ private function getNostoCategory()
     /**
      * Returns the HTML to render categories
      *
-     * @return MarkupableString
+     * @return string
      */
-    public function getAbstractObject()
+    public function toHtml()
```

Comment:
No need to override the `toHtml`. Keep the `getAbstractObject` method and return `$category`

Reply:
Reverted